### ISSUE TITLE
`filesystem error: cannot rename: Not a directory`

### ISSUE THREAD

**Issue by poperigby** [limo-app/limo#58]
I'm getting the following error when installing [this](https://www.nexusmods.com/skyrimspecialedition/mods/53878?tab=requirements&file_id=559137&nmm=1) mod:

```
[Error]: filesystem error: cannot rename: Not a directory [/home/cassidy/Games/Modding/Games/Skyrim/lmm_tmp_extract0/Backgrounds/shrooms/back.dds] [/home/cassidy/Games/Modding/Games/Skyrim/822/textures/terrain/blackreach/]
```

It seems to be happening with all of these Flat World Map Framework mods.

**Comment by limo-app** [comment]
Thanks for reporting this. It seems like this is the first mod author i found that does not specify the full target path for a file in the FOMOD xml and instead only gave the directory into which the file should be installed. This causes the standard libraries file move function to fail, which is something i did not know about.
I have fixed this in [165a438](https://github.com/limo-app/limo/commit/165a43836c08ec3e21819351fc95d5f298b1a5b0) and will push it as a minor update for the Flatpak version. It will probably take a few hours until the update goes live.

**Comment by poperigby** [comment]
Fixed. Thanks!
